Error in test_utils unittest

|Wazuh version|Component|
|---|---|
| master | Framework unittests |

## Description

Hi team!

It seems that there is some kind of error for the unit tests of the framework `utils.py` module. When running it alone, it works fine:
```
python3 -m pytest framework/wazuh/core/tests/test_utils.py --disable-warnings
================================================================== test session starts ===================================================================
platform linux -- Python 3.8.5, pytest-6.0.1, py-1.9.0, pluggy-0.13.1
rootdir: /home/selu/Git/wazuh/framework
plugins: trio-0.6.0, asyncio-0.14.0
collected 219 items                                                                                                                                      

framework/wazuh/core/tests/test_utils.py ......................................................................................................... [ 47%]
..................................................................................................................                                 [100%]

============================================================ 219 passed, 2 warnings in 0.85s =============================================================
```

However, when running it after any other test where `wazuh.core.utils` is imported, it fails as shown below:
```
python3 -m pytest framework/wazuh/core/tests/test_syscollector.py framework/wazuh/core/tests/test_utils.py --disable-warnings
================================================================== test session starts ===================================================================
platform linux -- Python 3.8.5, pytest-6.0.1, py-1.9.0, pluggy-0.13.1
rootdir: /home/selu/Git/wazuh/framework
plugins: trio-0.6.0, asyncio-0.14.0
collected 222 items                                                                                                                                      

framework/wazuh/core/tests/test_syscollector.py ...                                                                                                [  1%]
framework/wazuh/core/tests/test_utils.py ................................................................F........................................ [ 48%]
..................................................................................................................                                 [100%]

======================================================================== FAILURES ========================================================================
__________________________________________________________________ test_load_wazuh_xml ___________________________________________________________________

mock_compile = <MagicMock name='compile' id='140158566664272'>

    @patch('wazuh.core.utils.compile', return_value='Something')
    def test_load_wazuh_xml(mock_compile):
        """Test load_wazuh_xml function."""
        with patch('wazuh.core.utils.open') as f:
            f.return_value.__enter__.return_value = StringIO(test_xml)
            result = load_wazuh_xml('test_file')
    
>           assert isinstance(result, ElementTree.Element)
E           AssertionError: assert False
E            +  where False = isinstance(<Element 'root_tag' at 0x7f79359188b0>, <class 'xml.etree.ElementTree.Element'>)
E            +    where <class 'xml.etree.ElementTree.Element'> = ElementTree.Element

framework/wazuh/core/tests/test_utils.py:537: AssertionError
================================================================ short test summary info =================================================================
FAILED framework/wazuh/core/tests/test_utils.py::test_load_wazuh_xml - AssertionError: assert False
======================================================= 1 failed, 221 passed, 2 warnings in 0.79s ========================================================
```

The error needs to be investigated and fixed. 

## Checks

### wazuh/wazuh
- Unit tests without failures. Updated and/or expanded if there are new functions/methods/outputs:
  - [x] Cluster (`framework/wazuh/core/cluster/tests/` & `framework/wazuh/core/cluster/dapi/tests/`)
  - [x] Core (`framework/wazuh/core/tests/`)
  - [x] SDK (`framework/wazuh/tests/`)
  - [x] RBAC (`framework/wazuh/rbac/tests/`)
  - [x] API (`api/api/tests/`)
- API tavern integration tests without failures. Updated and/or expanded if needed (`api/test/integration/`):
  - [x] Affected tests 
  - [x] Affected RBAC (black and white) tests
- [x] Review integration test mapping using the script (`api/test/integration/mapping/integration_test_api_endpoints.json`)
- [x] Review of spec.yaml examples and schemas (`api/api/spec/spec.yaml`)
- [x] Review exceptions remediation when any endpoint path changes or is removed (`framework/wazuh/core/exception.py`)
- [x] Changelog (`CHANGELOG.md`)
<!-- If changes are made to any of the following components, uncomment the corresponding line 
- [ ] **Integration tests** without failures for API configuration (`/wazuh-qa/tests/integration/test_api/test_config/`)
- [ ] **System tests** for agent enrollment process (`/wazuh-qa/tests/system/test_cluster/test_agent_enrollment`)
- [ ] **System tests** for agent info sync process in cluster (`/wazuh-qa/tests/system/test_cluster/test_agent_info_sync`)
- [ ] **System tests** for agent key polling (`/wazuh-qa/tests/system/test_cluster/test_agent_key_polling`)
- [ ] **System tests** for JWT invalidation (`/wazuh-qa/tests/system/test_jwt_invalidation`)
-->

### wazuh/wazuh-documentation
- [x] Migration from 3.X for changed endpoints (`source/user-manual/api/equivalence.rst`)
- [x] Update RBAC reference with new/modified actions/resources/relationships (`source/user-manual/api/rbac/reference.rst`)

--- a/api/test/integration/test_experimental_endpoints.tavern.yaml
+++ b/api/test/integration/test_experimental_endpoints.tavern.yaml
@@ -551,13 +551,17 @@ stages:
             - <<: *ports_response
               agent_id: '001'
             - <<: *ports_response
+              agent_id: '001'
+            - <<: *ports_response
+              agent_id: '003'
+            - <<: *ports_response
               agent_id: '003'
             - <<: *ports_response
               agent_id: '005'
             - <<: *ports_response
               agent_id: '007'
           failed_items: []
-          total_affected_items: !anyint
+          total_affected_items: 6
           total_failed_items: 0
 
 ---

--- a/api/test/integration/test_experimental_endpoints.tavern.yaml
+++ b/api/test/integration/test_experimental_endpoints.tavern.yaml
@@ -557,7 +557,7 @@ stages:
             - <<: *ports_response
               agent_id: '007'
           failed_items: []
-          total_affected_items: 4
+          total_affected_items: !anyint
           total_failed_items: 0
 
 ---

--- a/framework/wazuh/core/utils.py
+++ b/framework/wazuh/core/utils.py
@@ -22,8 +22,7 @@ from pyexpat import ExpatError
 from shutil import Error, copyfile, move
 from subprocess import CalledProcessError, check_output
 from xml.dom.minidom import parseString
-from xml.etree import ElementTree as ET
-from xml.etree.ElementTree import fromstring
+from xml.etree import ElementTree
 
 import wazuh.core.results as results
 from api import configuration
@@ -737,7 +736,7 @@ def load_wazuh_xml(xml_path, data=None):
                '\n'.join([f'<!ENTITY {name} "{value}">' for name, value in custom_entities.items()]) + \
                '\n]>\n'
 
-    return fromstring(entities + '<root_tag>' + data + '</root_tag>')
+    return ElementTree.fromstring(entities + '<root_tag>' + data + '</root_tag>')
 
 
 class WazuhVersion:
@@ -1610,7 +1609,7 @@ def upload_file(content, path, check_xml_formula_values=True):
     def escape_formula_values(xml_string):
         """Prepend with a single quote possible formula injections."""
         formula_characters = ('=', '+', '-', '@')
-        et = ET.ElementTree(ET.fromstring(f'<root>{xml_string}</root>'))
+        et = ElementTree.ElementTree(ElementTree.fromstring(f'<root>{xml_string}</root>'))
         full_preprend, beginning_preprend = list(), list()
         for node in et.iter():
             if node.tag and node.tag.startswith(formula_characters):

